roles/contiv_cluster/tasks/main.yml

```diff
@@ -11,10 +11,16 @@
   tags:
     - prebake-for-dev
 
+- name: check for collins image
```

Comment:
this task will need `prebake-for-dev` as well so that it is run in build (and asnible sanity) otherwise `collins_exists` will not be registered and we will get variable undefined error